docs/content/config/best-practices/folders.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Best Practices | Folders'
+hide:
+  - toc # Hide Table of Contents for this page
+---
+
+## Add a folder
+
+Please refer to `target/dovecot/15-mailboxes.conf` for existing folders definition and potential folders creation.
+More detail available on [Upstream example config](https://github.com/dovecot/core/blob/master/doc/example-config/conf.d/15-mailboxes.conf).
+
+## Consideration
+
+### Create folder needed as soon as possible. 
+
+Different mail clients have different way of handling folders. Users might experience issues such like archived emails are only available in local environment. And if the folders are available / created later, users shall need to move quite a few emails between exising and newly created folders.
+
+
+### Tag v.s. Folder name
+
+Certain email clients simply ignore the tag defined in `RFC 6154` and look of the exact name instead. So it might be necessary to create folder with name the same as tag. 
+
+### i18n 
+
+Users and email clients might want to have folder names in local language instead of English. Please test carefully. This might need to find a balance with the tag v.s. folder name issue.
+
```

Comment:
```suggestion
## Adding a mailbox folder

See [`target/dovecot/15-mailboxes.conf`][gh-config-dovecot-mailboxes] for existing folder definitions you can modify or enable.
Dovecot also [provides their own example config][dovecot-config-mailboxes] for reference, with further information in [their documentation][dovecot-docs-mailboxes].

## Caution

### Adding folders to an existing setup

Handling of newly added mailbox folders can be inconsistent across mail clients:

- Users may experience issues such as archived emails only being available locally.
- Users may need to migrate emails manually between two folders.

### Tag Support

Not all mail clients support tags (_defined in [RFC 6154][rfc-6154]_), using the exact mailbox name instead. To support those clients, you may prefer to keep the tag and folder name identical.

### Internationalization (i18n)

Users and mail clients may prefer localized mailbox names instead of English. Take care to test localized names work well, keep in mind concerns such as tag support.

[gh-config-dovecot-mailboxes]: https://github.com/docker-mailserver/docker-mailserver/blob/master/target/dovecot/15-mailboxes.conf
[dovecot-config-mailboxes]: https://github.com/dovecot/core/blob/master/doc/example-config/conf.d/15-mailboxes.conf
[dovecot-docs-mailboxes]: https://doc.dovecot.org/configuration_manual/namespace/#mailbox-settings
[rfc-6154]: https://datatracker.ietf.org/doc/html/rfc6154

```

I have not read the RFC referenced, but it only has two `ctrl+f` matches for "tag" is that the correct word for that feature? Is this what a mail client refers to it as?

I also noticed the docs here focusing on "folder" name and creation, but is that the correct terminology or is it from the user perspective of what a mailbox is?

Honestly, I'm not sure about relevance for the i18n section? I had a brief search through some documents like Dovecot docs and the RFC mentioned, but I saw nothing about languages being mentioned. What is the actual feature or issue being described there? Is there an example scenario?